src/features/chart/components/Chart.tsx

```diff
@@ -17,33 +17,35 @@ import { ErrorMessage } from "../../error-message";
 import { AppState } from "store";
 import "./Chart.css";
 
-type Tenor = { value: string; label: string };
+export type ChartRange = "1D" | "5D" | "1M" | "1Y" | "5Y" | "MAX";
+
+type Tenor = { value: ChartRange };
 
 const tenors: Tenor[] = [
-  { value: "1d", label: "1D" },
-  { value: "5d", label: "5D" },
-  { value: "1m", label: "1M" },
-  { value: "1y", label: "1Y" },
-  { value: "5y", label: "5Y" },
-  { value: "max", label: "MAX" }
+  { value: "1D" },
+  { value: "5D" },
+  { value: "1M" },
+  { value: "1Y" },
+  { value: "5Y" },
+  { value: "MAX" }
```

Comment:
Q: Why are you using an array of objects rather than just an array? Do you need a value key?

Reply:
Good spot Dan, this could be changed to an array now

Reply:
Because previously we have both value and labels. 
A good point, will change it to just an array